### ISSUE TITLE
[QUESTION]  Status Indicators check for auth.mysite.com instead of photo.mysite.com

### ISSUE THREAD

**Issue by masterwishx** [Lissy93/dashy#713]
### Question

 Status Indicators check for auth.mysite.com instead of photo.mysite.com becose of authelia 
i understand that dashy need to render site for status , but uptimekuma somehow checks without it .....
is any way to check site like kuma make it ?
or some other steps needs to make ?

### Category

Status Checking

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [question hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Yeah, this has been discussed here: https://github.com/Lissy93/dashy/issues/616
But long story short, Uptime Kuma does not have an API, so it wouldn't be possbile.

But you could just write your own status check endpoint, then [point to it](https://github.com/Lissy93/dashy/blob/master/docs/status-indicators.md#using-a-different-endpoint) instead.
